job_bundles/tile_render_maya_ffmpeg/template.yaml

```diff
@@ -0,0 +1,331 @@
+  default: '<Scene>'
+- name: ImageWidth
+  type: INT
+  userInterface:
+    control: SPIN_BOX
+    label: Image Width
+    groupLabel: Maya Settings
+  minValue: 1
+  description: The image width of the output.
+- name: ImageHeight
+  type: INT
+  userInterface:
+    control: SPIN_BOX
+    label: Image Height
+    groupLabel: Maya Settings
+  minValue: 1
+  description: The image height of the output.
+- name: NumXTiles
+  type: INT
+  userInterface:
+    control: SPIN_BOX
+    label: Tiles across width
+    groupLabel: Tile Rendering Settings
+  minValue: 1
+  description: The number of tiles across the x axis.
+- name: NumYTiles
+  type: INT
+  userInterface:
+    control: SPIN_BOX
+    label: Tiles across height
+    groupLabel: Tile Rendering Settings
+  minValue: 1
+  description: The number of tiles across the y axis.
+- name: ArnoldErrorOnLicenseFailure
+  type: STRING
+  userInterface:
+    control: CHECK_BOX
+    label: Error on License Failure
+    groupLabel: Arnold Renderer Settings
+  description: Whether to produce an error when there is an Arnold license failure.
+  default: 'false'
+  allowedValues:
+  - 'true'
+  - 'false'
+- name: OverrideAdaptorWheels
+  type: PATH
+  objectType: DIRECTORY
+  dataFlow: IN
+  userInterface:
+    control: CHOOSE_DIRECTORY
+    label: Wheels directory
+    groupLabel: Override settings
+  description: A directory that contains wheels for openjd, deadline, and the overridden
+    adaptor.
+- name: OverrideAdaptorName
+  type: STRING
+  userInterface:
+    control: LINE_EDIT
+    label: Adaptor Name
+    groupLabel: Override settings
+  description: The name of the adaptor to override, for example NukeAdaptor or MayaAdaptor.
+  default: MayaAdaptor
+steps:
+- name: masterLayer tile render
+  parameterSpace:
+    taskParameterDefinitions:
+    - name: Frame
+      type: INT
+      range: '{{Param.Frames}}'
+    - name: Camera
+      type: STRING
+      range:
+      - RenderCam
+    - name: TileNumberX
+      type: INT
+      range: '1-{{Param.NumXTiles}}'
+    - name: TileNumberY
+      type: INT
+      range: '1-{{Param.NumYTiles}}'
+  stepEnvironments:
+  - name: Maya
+    description: Runs Maya in the background.
+    script:
+      embeddedFiles:
+      - name: initData
+        filename: init-data.yaml
+        type: TEXT
+        data: |
+          renderer: arnold
+          render_layer: masterLayer
+          scene_file: '{{Param.MayaSceneFile}}'
+          project_path: '{{Param.ProjectPath}}'
+          output_file_path: '{{Param.OutputFilePath}}'
+          render_setup_include_lights: {{Param.RenderSetupIncludeLights}}
+          strict_error_checking: {{Param.StrictErrorChecking}}
+          output_file_prefix: '{{Param.OutputFilePrefix}}'
+          image_width: {{Param.ImageWidth}}
+          image_height: {{Param.ImageHeight}}
+          error_on_arnold_license_fail: {{Param.ArnoldErrorOnLicenseFailure}}
+      actions:
+        onEnter:
+          command: MayaAdaptor
```

Comment:
Could you switch this to the command `maya-openjd`?  We added the replacement name, but haven't done a full sweep to finish that.